src/Api/Parser/AbstractRestParser.php

```diff
@@ -93,6 +93,12 @@ private function extractHeader(
     ) {
         $value = $response->getHeaderLine($shape['locationName'] ?: $name);
 
+        if (!empty($shape->offsetGet('jsonvalue'))
```

Comment:
I just wanted to make sure that the value is properly checked. WE have had bugs before where a value was set to empty strings and it resolved as true. I believe it never hurts to be too careful.